include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -15,26 +15,29 @@ namespace asset
 {
 
 // TODO: move this to appropriate class
-enum E_DESCRIPTOR_TYPE : uint32_t
+enum E_DESCRIPTOR_TYPE : uint8_t
```

Comment:
move to `IDescriptor` class and rename to `E_TYPE`, can make it a enum-class while you're at it

Reply:
Resolved.